app/views/metadata_presenter/component/_autocomplete.html.erb

```diff
@@ -0,0 +1,14 @@
+<%=
+  f.govuk_collection_select component.id.to_sym,
+  component.items,
+  :id,
+  :name,
+  label: { text: input_title },
+  hint:  {
+      data: { "fb-default-text" => default_text('hint') },
+      text: component.hint
+    },
+  style: "width:50%"
```

Comment:
not sure we need this class

Reply:
I've removed the inline styling and added the design system class